### ISSUE TITLE
TalkBack pressing volume buttons insists on changing notification volume and not music volume.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#312]
```
What steps will reproduce the problem?
1. With TalkBack enabled, attempt to turn the volume up or down.
2. Listen until you hear that the notification volume has been set to x% where 
X = the percentage of volume.
3. Get extremely frustrated and roll back your version of talkback.  This 
happens with the latest stable version as well, not just this beta.

What is the expected output? What do you see instead?
expected output = music volume set to X%.  See instead:  Notification volume 
set to X%.

What version of the product are you using? On what operating system?
3.5 beta 1 on Android 4.3 

Please provide any additional information below.
Nexus 7 second generation

```

Original issue reported on code.google.com by `TonyHSpe...@gmail.com` on 18 Aug 2013 at 11:50

**Comment by GoogleCodeExporter** [comment]
```
Hi,

Apologize that I am not one of the Google devs but I hope to share some words 
in this issue.

I don't think this is a bug in TalkBack, but this is a new approach in the 
Google Accessibility framework API for Android 4.3. This is to standardize the 
way to use accessibility by adding one more finger to use the accessibility 
features. For example, for a sighted, to scroll a list, they use one finger but 
for TalkBack user, we must use two finger instead. So, to adjust TalkBack 
volume, we must hold one finger on the screen prior to press the volume keys.
```

Original comment by `andr...@smartphone.my` on 19 Aug 2013 at 8:10

**Comment by GoogleCodeExporter** [comment]
```
The behavior as noted in #1 is correct for phones.  However, for tablet 
devices, the default media stream is the music stream, not the 
notification/ringer stream.  TalkBack has the notification/ringer stream hard 
coded as the stream to change when the volume keys are pressed with no finger 
on the screen.  This should be switched to use 
AudioManager.USE_DEFAULT_STREAM_TYPE so that on tablet devices the correct 
fallback stream is used.
```

Original comment by `caseybur...@google.com` on 5 Dec 2013 at 7:50
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 3.5.1
```

Original comment by `caseybur...@google.com` on 6 Dec 2013 at 3:28
- Changed state: **Fixed**
